Makefile

```diff
@@ -65,3 +64,15 @@ runserver:
 css:
 	$(VENV); gulp sass
 	$(VENV); gulp watch
+
+##### DOCKER INTEGRATION
+##### required docker-compose http://docs.docker.com/compose/install/
+
+docker:
+	docker-compose build
+	docker-compose up -d
+	docker-compose run web src/manage.py migrate
+	docker-compose run web src/manage.py createsuperuser --username=admin --email=admin@example.com
+	docker-compose ps
```

Comment:
I actually want to print out if all containers are running for feedback, thats intentionally